### ISSUE TITLE
TypeError: o.toFixed is not a function

### ISSUE THREAD

**Issue by JanithaR** [jhen0409/react-native-debugger#367]
I know this is very similar to [Console thrown a lot of errors when I use React Inspector on Android](https://github.com/jhen0409/react-native-debugger/blob/master/docs/troubleshooting.md#console-thrown-a-lot-of-errors-when-i-use-react-inspector-on-android
) but my `react-devtools-core` version is 3.6.0

The error happens exactly the same way as show in this issue https://github.com/jhen0409/react-native-debugger/issues/84#issuecomment-304611817
I try to inspect elements and errors are thrown.

React Native Debugger app version: 0.9.10
React Native version: 0.55.3
Platform: iOS
Is real device of platform: No
Operating System: macOS

![Screen Shot 2019-07-11 at 11 33 52 AM](https://user-images.githubusercontent.com/3599872/61025978-d6dee380-a3cf-11e9-9114-6f8f88d86964.png)

**Comment by JanithaR** [comment]
Figured out the problem. It's this styling,
```
container: {
        height: 60,
        paddingLeft: '5%',
        paddingTop: 10,
        paddingRight: '5%',
        paddingBottom: 10,
        flex: 1,
        justifyContent: 'center'
    }
```
Commenting out `paddingLeft: '5%',` and `paddingRight: '5%',` does not cause any errors.

**Comment by jhen0409** [comment]
It should be report into [`facebook/react-devtools`](https://github.com/facebook/react-devtools/issues).

Also, It may have been fixed in #371, if you are interested in, you can follow https://github.com/jhen0409/react-native-debugger/pull/371#issuecomment-513513181 to try.
